test/functional/verify/latest-missing/lines-checked

```diff
@@ -1,2 +1,2 @@
-Unable to get latest version for install
-Error: Fix did not fully succeed
+Curl error: (37) Couldn't read a file:// file
+Failed verify initialization, exiting now.
```

Comment:
Is this an expected change in test output?

Reply:
yes